opm/parser/share/keywords/000_Eclipse100/O/OPERNUM

```diff
@@ -0,0 +1,5 @@
+{
+ "name" : "OPERNUM",
```

Comment:
@andlaus we don't support the `OPERNUM` keyword, but I added it to the `REGIONS` section in  #1031